Dockerfile

```diff
@@ -36,4 +40,6 @@ COPY qgis.desktop ${DESKTOP_FILES_DIR}/qgis.desktop
 # See: https://manpages.ubuntu.com/manpages/focal/en/man1/xdg-autostart.1.html
 COPY qgis.desktop /etc/xdg/autostart/qgis.desktop
 
+COPY qgis.bash /home
```

Comment:
@Rub21 - when the container is run on JupyterHub, the home directory in the container will be swapped out for the user's home directory on the Hub. Therefore, we should not copy or use any files in `/home` - can you just copy and run this file anywhere else on the system other than `/home` - just creating a new directory or putting it in `/srv` or so should be fine.